instant-xml-macros/src/de.rs

```diff
@@ -207,15 +246,56 @@ impl Deserializer {
         let field_var_str = field_var.to_string();
         let const_field_var_str = Ident::new(&field_var_str.to_uppercase(), Span::call_site());
         let field_type = match &field.ty {
-            syn::Type::Path(v) => v.path.get_ident(),
+            syn::Type::Path(v) => match v.path.get_ident() {
+                Some(ident) => ident.into_token_stream(),
+                None => {
+                    if v.path.segments.is_empty() {
+                        panic!("Wrong declaration");
+                    };
+
+                    // Cow<>, Option<>
+                    let mut out =
+                        (&v.path.segments.first().expect("type name").ident).into_token_stream();
+                    match &(v.path.segments.first().expect("type name")).arguments {
```

Comment:
So what problem is this trying to solve?

Reply:
I guess my question is, why do we need to do that?

Reply:
After some experimentation, I couldn't get the lifetimes to work. I rewrote the code to discard lifetimes to be a little more robust, seems to be working for now.